index.bs

```diff
@@ -355,6 +355,73 @@ User agents may use a [=new information about the user's intent|number of criter
 to grant access to the [=sensor readings|readings=].
 Access may be granted without prompting the user.
 
+<h3 id="mitigation-strategies-case-per-case">Mitigation strategies applied on a case per case basis</h3>
+
+Each [=sensor type=] will need to be assessed individually
+taking in account the use cases it enables
+and its particular threat profile.
+While some of the below mitigation strategies
+are effective for certain sensors,
+they might also hinder or altogehter prevent certain use cases.
+
+
+<h4 id="limit-max-freqency" dfn>Limit maximum polling frequency</h4>
```

Comment:
It is matter of taste or document style. I would  add note to the same section, that UA could stop sensors as a mitigation strategy, same as limiting frequency to 1.0E-15 :D